src/snowflake/cli/_plugins/nativeapp/application_package_entity.py

```diff
@@ -723,6 +748,116 @@ def check_index_changes_in_git_repo(
+        cls,
+        console: AbstractConsole,
+        project_root: Path,
+        deploy_root: Path,
+        bundle_root: Path,
+        generated_root: Path,
+        artifacts: list[PathMapping],
+        package_name: str,
+        package_role: str,
+        package_distribution: str,
+        version: Optional[str],
+        force: bool,
+        interactive: bool,
+    ):
+        """
+        Drops a version defined in an application package. If --force is provided, then no user prompts will be executed.
+        """
+        is_interactive = False
+        if force:
+            policy = AllowAlwaysPolicy()
+        elif interactive:
+            is_interactive = True
+            policy = AskAlwaysPolicy()
+        else:
+            policy = DenyAlwaysPolicy()
+
+        # 1. Check for existing an existing application package
+        show_obj_row = cls.get_existing_app_pkg_info(package_name, package_role)
+        if not show_obj_row:
+            raise ApplicationPackageDoesNotExistError(package_name)
+
+        # 2. Check distribution of the existing application package
+        actual_distribution = cls.get_app_pkg_distribution_in_snowflake(
+            package_name, package_role
+        )
+        if not cls.verify_project_distribution(
+            console=console,
+            package_name=package_name,
+            package_role=package_role,
+            package_distribution=package_distribution,
+            expected_distribution=actual_distribution,
+        ):
+            console.warning(
+                f"Continuing to execute version drop on application package "
+                f"{package_name} with distribution '{actual_distribution}'."
+            )
+
+        # 3. If the user did not pass in a version string, determine from manifest.yml
+        if not version:
+            console.message(
+                dedent(
+                    f"""\
+                        Version was not provided through the Snowflake CLI. Checking version in the manifest.yml instead.
+                        This step will bundle your app artifacts to determine the location of the manifest.yml file.
+                    """
+                )
+            )
+            cls.bundle(
+                project_root=project_root,
+                deploy_root=deploy_root,
+                bundle_root=bundle_root,
+                generated_root=generated_root,
+                artifacts=artifacts,
+                package_name=package_name,
+            )
+            version, _ = find_version_info_in_manifest_file(deploy_root)
+            if not version:
+                raise ClickException(
+                    "Manifest.yml file does not contain a value for the version field."
+                )
+
+        # Make the version a valid identifier, adding quotes if necessary
+        version = to_identifier(version)
+
+        console.step(
+            f"About to drop version {version} in application package {package_name}."
```

Comment:
did this message change? was there a backslash at the beginning?

Reply:
it was a multiline string when it didn't need to be since it was only one line